File: Python/dia_06.py
# Dia 6 = Crie um programa que determine se um número é par ou ímpar.

def par_impar(num = 0):
    if num == 0:
        return 'Par (0 é considerado um número par)'
    elif num % 2 == 0:
        return 'Par'
    else:
        return 'Ímpar'
        
resultado = par_impar()
print(resultado)

File: JavaScript/dia_06/dia_06.js
// Dia 6 = Crie um programa que determine se um número é par ou ímpar.

let divres = document.querySelector('div#res')

function par_impar() {
    let valor = Number(document.querySelector("input#numero_01").value)

    if (valor == 0) {
        divres.innerHTML = 'Par (0 é considerado um número par)'
    } else {
        if (valor % 2 === 0) {
            divres.innerHTML = `O valor ${valor} é Par`
        } else {
            divres.innerHTML = `O valor ${valor} é Ímpar`
        }
    }
}